### DISCUSSION TITLE
Read it later

### DISCUSSION THREAD

**Discussion by idoshamun** [dailydotdev/daily#364] [Feature Requests]
I think that on top of the existing bookmark feature there's room for a "Read it later" list.

As a user, I want to mark an article as "read it later" and it will be shown again in my feed the next time I open the feed even though it may be old or rank lower. Once I click on it, it will be removed from the "read it later" list.

What's the difference between bookmark and "read it later"?
Bookmarks can remain forever in the list and can be thought of as favorites, and "read it later" is an inbox zero kind of list.

Submitted on behalf of @arklev

**Comment by GoodnessEzeh** [comment]
That would be a good idea. Once bookmarked, an article remains on the list even after being read over and over again until it is marked otherwise, but on "Read it later", the article leaves the list when read only once.

**Comment by bgalvao** [comment]
_Adapted from https://github.com/dailydotdev/daily/discussions/478 opening comment_

I agree. This extension is somewhat disruptive: I am in the middle of work and open a new tab in the browser. Then I see that shiny title that really makes me want to read an article, but! it is not the right time as I am in the middle of a task.

Some thoughts on why this would be a good feature, based on personal experience:

- I've been abusing the bookmark feature, but then I end up with a collection of articles I haven't even read or checked whether they're "bookmark-quality".
- the user could relax from FOMO and only open an article when taking a break.
    - it has happened to me that I missed an article from the page upon refreshing dailydev's new tab.
- I don't like to have tabs opened from daily dev waiting for me and cluttering my view and still forgetting about them anyways, so this would be another reason to have a queue feature: to keep articles I want to read in a safe and tidy place.
- you could even collect and display a metric: how many users have queued a given article vs how many have upvoted it. This would capture the aspect of which titles get the attention of readers and which are actually valued by upvotes.

**Comment by bgalvao** [comment]
can we add tags to organize Read It Later and Bookmarks?

**Reply by idoshamun** [reply]
Can you elaborate please?

**Reply by nathanbalthazar** [reply]
> Can you elaborate please? 

I would love to save/organize the saved articles like this:

![image](https://user-images.githubusercontent.com/80779259/158000823-c2f63b23-353a-4983-8ebe-d81bfaaae048.png)
